app/src/lib/shells/win32.ts

```diff
@@ -12,11 +11,19 @@ export enum Shell {
   PowerShellCore = 'PowerShell Core',
   Hyper = 'Hyper',
   GitBash = 'Git Bash',
+  WslBash = 'WSL Bash (distro)',
+  WslBashDefault = 'WSL Bash (Default)',
 }
 
-export const Default = Shell.Cmd
+const WslShells: Array<IFoundShell<Shell>> = []
+
+export const Default: string = Shell.Cmd
```

Comment:
This was a remnant of something i tried earlier and stupidly forgot to roll back the changes - will do so now.

Reply:
I do believe this was done to ensure the drop down for the list of shells selects the correct index for each of the custom WSL distros. Each custom distro name will use the same enum identifier string and select the index associated to that. Eg.

Git Bash ==> 'Git Bash'   (index 0)
WSL (Ubuntu) ==> 'WSL Bash (distro)' (index 1)
WSL (MyDistro) ==> 'WSL Bash (distro)' (index 2)

If i've saved MyDistro - reloading the app will end up selecting index 1 instead of 2.

Happy to come up with another approach - i saw that the enum was simply a string and thought exact matching on the string would be more reliable and simpler than matching on a strongly typed enum. 

Happy to alter if there is a better approach that im not seeing.

Reply:
Thanks for the context. I'll think on this some more and see if I have any bright ideas.

Reply:
Yea i was unhappy swapping them from Shell to string. I might have a play around this weekend and see if I can get around that bug another way.

Reply:
I am wondering about storing alongside the shell selection in IAppState, a shellMetaData variable (string?), providing a postShellProcessing() method on shared.ts to use to deduce the specific wsl distro for win32 specifically, if linux/darwin have further complicated shells they can implement the pre and post shell processing methods as a future proofing type of thing. Thoughts?